### PR TITLE
CREATE: New document for documentation process

### PR DESCRIPTION
**Pull request by sksingh2022**
# Description

Updating documentation for the documentation.md file. Added content outlining the documentation processes and the information corresponds with the presentation by Talia on documentation ups killing

Fixes # (issue)

No fixes. Just adding content into the company handbook

- [x] Documentation (new)

# How Has This Been Tested?

Not applicable, this is a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by ghost** [comment]
What is this document for? 
The PR template has not been filled out?

**Comment by ghost** [comment]
I would consider making this a draft PR, and add content before you merge the blank file.

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
 - Processes
   - Developer Operations
   - [Quality Assurance](docs/processes/quality-assurance/quality-assurance-overview.md)
-  - Documentation
+  - [Documentation] (docs/processes/documentation/documentation.md)
```
Comment (ghost) [inline README.md]:
```suggestion
  - [Documentation](docs/processes/documentation/documentation.md)
```
Not sure if the space will affect anything technically, but for consistency I would remove it.

**Comment by ghost** [comment]
@sksingh2022, there are now two PRs which will likely create a merge conflict? Would you like to close one of the PRs?

**Comment by sksingh2022** [comment]
Hi talia, sorry about this, please see updated.

**Review by ghost — CHANGES_REQUESTED** [review]

File: docs/processes/documentation/documentation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+Documentation is one of the most important parts of any software epic. It is imperative to remember, you will not be working on the capstone projects forever and that there will be people taking your place within Thoth Tech.
```
Comment (ghost) [inline docs/processes/documentation/documentation.md]: I would prefer if our handbook was less capstone related and more company focused.
Although, Thith Tech is for capstone, it is still a viable open source company and should be spoken about as such.

Consider rephrasing: 

```suggestion
Thoth Tech prides itself on our ability to produce sustainable solutions, this can be achieved through documentation.
Documentation is an important element in the software development life cycle. It continues throughout the entire project.
It is imperative to remember, others will take over your projects or want to learn more about how it works and why. 
```

File: docs/processes/documentation/documentation.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Documentation is the offical information that is provided by the company for users and contributors. Documentation should provide clarity around troubleshooting and provide users and those involved with a basic idea of what it is the platform does and how it works.
```
Comment (ghost) [inline docs/processes/documentation/documentation.md]: Hopefully it is more than a basic understanding 


```suggestion
Documentation is offical information, provided by the company for users and contributors. Documentation is written for specific audiences and provide knowledge on the platform.
```

File: docs/processes/documentation/documentation.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+The Thoth Tech documentation should all be stored in the documentation repositry within the company GitHub. All documentation should be stored in a .md format and anything to do with research, meeting minutes and updates, should be stored within the [documentation repositry](https://github.com/thoth-tech/documentation).
+
+## Why is it important?
+
+Without documentation, there is no record of:
```
Comment (ghost) [inline docs/processes/documentation/documentation.md]: Not entirely true, it is possible to find a record but it is difficult.

I would remove this section and the following list. Easier to link to find information in one spot such as the Docs repo.

File: docs/processes/documentation/documentation.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+When documenting, we recommend you consider the following when writing your documentation:
+
+- Clear
+- Useful
+- Friendly
+- Appropriate
+- Consistent
+- Accessible
+
+Further to this, we ask you consider reading the documentation upskilling guide on the [training centre](docs/../../../learning/training/writing-documentation.md)
```
Comment (ghost) [inline docs/processes/documentation/documentation.md]: As this is essentially an index page that might have future links, I would consider a list. 


```suggestion
For more information:

- [writing documentation guide](docs/../../../learning/training/writing-documentation.md)
```

File: docs/processes/documentation/documentation.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Contributing
+
+Please see the [contribution guidelines](docs/processes/../../../quality-assurance/git-contribution-guide.md)
```
Comment (ghost) [inline docs/processes/documentation/documentation.md]: This should be linked to in documentation repo readme.md. 

As it is already mentioned in the handbook on the readme.md I would refrain from linking to it again.

File: docs/processes/documentation/documentation.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+Without documentation, there is no record of:
+
+- Choices and Decisions
+- Why functionality is added
+- What processes are
+- How users use the tool
+- What project updates have been added
+- And more
+
+Thoth Tech documentation are not only used by the one respective team. The documentation can be accesed at anytime by any of the Thoth Tech company members just to see where a project is heading and what progress has been made. Additionally, should other teams wish, they may collaborate and think of intuitve ways in which to further improve the projects.
```
Comment (ghost) [inline docs/processes/documentation/documentation.md]: I personally would not include the section about collaboration, I do not think this is the right place to suggest such activity and will depend on the area leads of that tri. Better to leave that go future area leads and have one source of truth.

```suggestion
Thoth Tech documentation is accessible to all.
```

**Review by ghost — CHANGES_REQUESTED** [review]
Thank you @sksingh2022 for your initiative creating this document. I have left some suggestions for you to consider that might help with the flow between other documents in the handbook.
Great job. Would be nice to see a similar message in the Docs repo readme.md

File: docs/processes/documentation/documentation.md (hunk 4 above)
Comment (sksingh2022) [reply docs/processes/documentation/documentation.md]: Section removed

File: docs/processes/documentation/documentation.md (hunk 6 above)
Comment (sksingh2022) [reply docs/processes/documentation/documentation.md]: Section also removed - thank you

**Comment by sksingh2022** [comment]
Hi Talia, I have successfully updated the markdown file and implemented your suggestions. Thank you for your support.

**Comment by ghost** [comment]
@tancnle is there a way to only merge some files and not others?
This PR has added unwanted changes to the package file and prettier file.

**Comment by tancnle** [comment]
> @tancnle is there a way to only merge some files and not others? This PR has added unwanted changes to the package file and prettier file.

@talzid We have to check out this branch, remove the unnecessary changes and do another commit. I can do that if there is no objection.

**Comment by ghost** [comment]
> > @tancnle is there a way to only merge some files and not others? This PR has added unwanted changes to the package file and prettier file.
> 
> @talzid We have to check out this branch, remove the unnecessary changes and do another commit. I can do that if there is no objection.

Please do! Could you also explain why this may have happened ?

**Comment by tancnle** [comment]
> Please do! Could you also explain why this may have happened?

The change might be made manually or it was a result of an unclean merge from the `main` branch. Looking at the history, it seems like the latter.

File: docs/processes/documentation/documentation.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
+## Introduction
```
Comment (tancnle) [inline docs/processes/documentation/documentation.md]: **question:** @talzid @sksingh2022 What is the difference between this doc and the [one](https://github.com/thoth-tech/handbook/blob//docs/documentation/index.md) we currenttly have in this folder?

File: README.md [hunk 9]
```diff
@@ -31,7 +31,7 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
 - Processes
   - Developer Operations
   - [Quality Assurance](docs/processes/quality-assurance/quality-assurance-overview.md)
-  - [Documentation](docs/documentation/index.md)
+  - [Documentation](docs/processes/documentation/documentation.md)
```
Comment (tancnle) [inline README.md]: **issue:** Is this change intended? We already had the file `docs/documentation/index.md` in the `main` branch. This change will effectively remove its reference from this file.

Comment (ghost) [reply README.md]: As per above, I was going to merge the files into one once merged and keep the index name for consistency. But did not want to 'rob' Shivam of the commit evidence

Comment (tancnle) [reply README.md]: > But did not want to 'rob' Shivam of the commit evidence

Gotcha. It is not really robing by the way. Shivam's change is still presented in this PR. We don't want to merge changes that are conflicting or duplicating.

By the way, I have removed the prettier and package changes in this PR. Please proceed with updating the documentation references and content whenever you see fit.

**Review by ghost — COMMENTED** [review]
Changes made and file merge confirmed

**Review by ghost — APPROVED** [review]

**Comment by sksingh2022** [comment]
Thank you @tancnle & @talzid for your help!
